packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -38,16 +39,13 @@ class FigureCanvasHTMLCanvas(FigureCanvasWasm):
     def __init__(self, *args, **kwargs):
         FigureCanvasWasm.__init__(self, *args, **kwargs)
 
-        # A count of the fonts loaded. To support testing
-        window.font_counter = 0
-
     def create_root_element(self):
         root_element = document.createElement("div")
         document.body.appendChild(root_element)
         return root_element
 
     def get_dpi_ratio(self, context):
-        if hasattr(window, "testing"):
+        if os.environ.get("TESTING_MATPLOTLIB"):
```

Comment:
is `TESTING_MATPLOTLIB` ever set?

Reply:
Oh sorry, the diff for test_matplotlib was collapsed so when I searched for it I didn't see it.